app/scripts/modules/core/src/search/searchResult/Renderers.tsx

```diff
@@ -105,3 +108,25 @@ export class AccountCell extends React.Component<ICellRendererProps> {
     );
   }
 }
+
+export class HealthCountsCell extends React.Component<ICellRendererProps> {
+  private cellContent(instanceCounts: IInstanceCounts) {
+    switch (instanceCounts) {
+      case undefined:
+        return <Spinner size="small"/>;
+      case null:
+        return <span className="error">error</span>;
```

Comment:
this could be better.

Reply:
Maybe just `?`? Seeing a column of `error`s might be alarming...